templates/node--16.html.twig

```diff
@@ -1,5 +1,3 @@
-{{ attach_library('eirh_subtheme/eventshub') }}
```

Comment:
Attaching the library via preprocess hook now. Although, this might still work now that I've found the DL conflict.

Reply:
https://stanfordits.atlassian.net/browse/EIRH-81